docs/team/jyrw.md

```diff
@@ -5,18 +5,147 @@ title: Joshua's Project Portfolio Page
 
 ### Project: Dash
 
-Given below are my contributions to the project.
+Dash is a desktop application that acts as a Dashboard for managing your contacts and tasks. 
+Dash operates using a CLI (Command Line Interface) but has a GUI made with JavaFX. 
+It is written in Java, and has about 11 kLoC.
 
-* **Code contributed**: [RepoSense link]()
+Given below are my contributions to the project.
 
-* **Project management**:
+* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2122s1.github.io/tp-dashboard/?search=&sort=groupTitle&sortWithin=title&since=2021-09-17&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=false&tabOpen=true&tabType=authorship&tabAuthor=jyrw&tabRepo=AY2122S1-CS2103T-W15-2%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false)
```

Comment:
Just a minor thing but perhaps you can consider obtaining a shorter URL by going on the tp dashboard and just searching for your Github user name instead, it will make the URL a lot shorter if you grab the link to your code contribution that way!